crates/noirc_evaluator/src/brillig/acvm_brillig.rs

```diff
@@ -0,0 +1,155 @@
+use acvm::FieldElement;
+//THIS IS A TEMPORARY FILE, all warnings are disabled.
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all, dead_code)]
+
+pub type Label = usize;
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all)]
+pub type BrilligOpcode = Opcode;
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all, dead_code)]
+pub type BrilligType = Typ;
+#[allow(clippy::all, unreachable_code, unreachable_pub, rustdoc::all, dead_code)]
+pub type BrilligValue = Value;
```

Comment:
Is this PR meant to include the brillig PR? Or was it supposed to be merged somewhere other than master?

Reply:
I think you're commenting on a stale diff prior to merging from master...

Reply:
Indeed I was, thank you 🙂